src/jmh/java/org/commcare/LoadAndInitLargeForm.java

```diff
@@ -0,0 +1,53 @@
+package org.commcare;
+
+import org.commcare.session.SessionNavigator;
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.Group;
+import org.openjdk.jmh.runner.Runner;
+import org.openjdk.jmh.runner.RunnerException;
+import org.openjdk.jmh.runner.options.Options;
+import org.openjdk.jmh.runner.options.OptionsBuilder;
+
+import org.commcare.modern.session.SessionWrapper;
+import org.commcare.test.utilities.MockApp;
+import org.commcare.test.utilities.MockSessionNavigationResponder;
+import org.javarosa.form.api.FormEntryController;
+
+public class LoadAndInitLargeForm {
+
+    @Benchmark()
+    @Group("form_init")
+    public void recForm() throws Exception {
```

Comment:
Does profiling for this method already automatically occur when the program is run because of the `@Benchmark()` tag? If so, where would it be reported to? If not, do you know what needs to be added to do so?